.vscode/launch.json

```
@@ -6,16 +6,16 @@
       "type": "node",
       "request": "launch",
       "cwd": "${workspaceRoot}",
-      "runtimeExecutable": "yarn",
-      "runtimeArgs": ["debug"],
-      "port": 5858
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run", "debug"],
+      "runtimeVersion": "20.18.0"
```

Could we be less strict on the version?
```suggestion
      "runtimeVersion": "20"
```